actions/check_if_message_MOD.js

```diff
@@ -19,17 +19,16 @@ module.exports = {
     return `
 <div>
   <message-input dropdownLabel="Source Message" selectId="message" variableContainerId="varNameContainer" variableInputId="varName"></message-input>
+  <br><br><br>
 
-  <br><br><br><br>
   <div style="float: left; width: 40%;">
     <span class="dbminputlabel">Check If Message</span>
     <select id="info" class="round">
       <option value="0">Is Pinnable?</option>
       <option value="1">Is Pinned?</option>
       <option value="2">Is Deletable?</option>
-      <option value="3">Is Deleted?</option>
       <option value="4">Is TTS?</option>
-      <option value="5">Is Of Discord?</option>
+      <option value="5">Is From Discord?</option>
       <option value="6">Includes @everyone Mention?</option>
```

Comment:
should add `builkDeletable`
```suggestion
      <option value="6">Includes @everyone Mention?</option>
      <option vlaue="7">Is Bulk Deleteable?</option>
```

Reply:
Added + more options